src/main/java/com/gregtechceu/gtceu/data/lang/LangHandler.java

```diff
@@ -1087,7 +1087,9 @@ public static void init(RegistrateLangProvider provider) {
         provider.add("gtceu.multiblock.not_enough_energy", "WARNING: Machine needs more energy.");
         provider.add("gtceu.multiblock.not_enough_energy_output", "WARNING: Energy Dynamo Tier Too Low!");
         provider.add("gtceu.multiblock.waiting", "WARNING: Machine is waiting.");
-        provider.add("gtceu.multiblock.batch_enabled", "Batching Mode: Enabled (%sx)");
+        provider.add("gtceu.multiblock.total_runs", "Performing %d Recipes Together");
```

Comment:
Stylistic thing, not super important:
```suggestion
        provider.add("gtceu.multiblock.total_runs", "Performing %d Recipes in Parallel");
```
I think keeping this wording for the number of total runs is better, as the recipes *are* being run in parallel, even for batch mode, but you could also word it as `Performing x Recipes at once`.